observability/WFLY-14854_add_opentelemetry_support.adoc

```diff
@@ -0,0 +1,76 @@
+ = Add OpenTelemetry Tracing Support to WildFly
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+https://opentelemetry.io[OpenTelemetry] is an ambitious specification that aims to encapsulate APIs for tracing, metrics, and logging. With OpenTracing (which WildFly already supports) having shutdown in favor of OpenTelemetry, we need to migrate our support as well. This project will only cover the tracing aspects of OpenTelemetry. Any future support for the metrics and logging parts of the OpenTelemetry spec will need to be handled by a separate effort.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-14854
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1688
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+* [X] Engineering
+* [X] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+* [ ] Managed domain
+* [x] OpenShift s2i
+* [x] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
+
+* The server/domain administrator should be able to configure OpenTelemetry export and span processor options
+* Jakarta REST endpoints should be automatically traced
+* Adherence to the Observability Standard for Runtimes
+* Context propagation should work for both incoming and outgoing request (e.g., a `ClientRequestFilter`)
+
+=== Nice-to-Have Requirements
+
```

Comment:
What is the final scope? Will any of the nice-to-have requirements be implemented? Will there be follow-up RFEs for unimplemented requirements?

Reply:
There are no plans to implement these, and I'm inclined to wait to file RFEs until we see if the community (or downstream EAP customers) show interest in them.